kotest-property/build.gradle.kts

```diff
@@ -0,0 +1,96 @@
+plugins {
+   id("java")
+   id("kotlin-multiplatform")
+   id("java-library")
+}
+
+repositories {
+   mavenCentral()
+}
+
+kotlin {
+
+   targets {
+      jvm {
+         targets {
+            jvm {
+               compilations.all {
+                  kotlinOptions {
+                     jvmTarget = "1.8"
+                  }
+               }
+            }
+         }
+      }
+      js {
+         val main by compilations.getting {
+            kotlinOptions {
+               moduleKind = "commonjs"
+            }
+         }
+      }
+   }
+
+   targets.all {
+      compilations.all {
+         kotlinOptions {
+            freeCompilerArgs + "-Xuse-experimental=kotlin.Experimental"
+         }
+      }
+   }
+
+   sourceSets {
+
+      val commonMain by getting {
+         dependencies {
+            implementation(kotlin("stdlib-common"))
+            api(project(":kotest-assertions"))
+            implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-common:1.3.3")
+            implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3")
+         }
+      }
+
+      val jsMain by getting {
+         dependsOn(commonMain)
+         dependencies {
+            implementation(kotlin("stdlib-js"))
+         }
+      }
+
+      val jvmMain by getting {
+         dependsOn(commonMain)
+         dependencies {
+            implementation(kotlin("stdlib-jdk8"))
+            implementation(kotlin("reflect"))
+            implementation("com.github.wumpz:diffutils:2.2")
+            implementation("com.univocity:univocity-parsers:2.8.3")
+            api("io.arrow-kt:arrow-core:0.10.3")
```

Comment:
Do we need to include arrow as an api? Could it be implementation?

Reply:
It will be removed when I remove arrow out of the main modules, there's another ticket for it.